lms/templates/wiki/plugins/attachments/index.html

```diff
@@ -87,7 +87,7 @@ <h3>
               <td class="attachment-actions">
                 {% if attachment|can_write:user %}
                   {% if not attachment.current_revision.deleted %}
-                {% if 'attachment.article = article' %}
+                {% if attachment.article == article %}
```

Comment:
Was this the equivalent of `{% if 'string' %}` before the change, and hence always `True`?

Reply:
By my read of the template format and the code - yes. Any issues with approval given this?

Reply:
We discussed this offline. This template line concerned wiki attachments which we don't even support in edx-platform.